### DISCUSSION TITLE
Share Your Experience with litgen in Your Projects!

### DISCUSSION THREAD

**Discussion by pthom** [pthom/litgen#19] [Announcements]
Whether you’re just starting with litgen or have been using it for a while, we’d love to hear how it's been helpful in your projects! This is a space where you can share your experiences, challenges, and insights with the community.

Some ideas to get started:

* What kind of project are you using litgen for?
* How has litgen helped you with your Python bindings?
* Any advices for other users?

By sharing your experiences, you not only help us understand how litgen is being used but also inspire others in the community who might be working on similar projects. Plus, your feedback is invaluable in shaping the future of litgen! 🚀

Feel free to start a new discussion or comment below. Looking forward to hearing from you!

**Comment by asafij1111** [comment]
Is there currently a parsing bug in the header_amalgamation script? I keep having the same problem where it cannot include standard library includes because it is removing the closing ">".
Example:  FileNotFoundError: #include <iostream

**Comment by pthom** [comment]
Please share a minimal reproducible example

**Reply by asafij1111** [reply]
I fixed the problem by using  options.local_includes_startwith so that it would not include any system includes. Some documentation that this is necessary to avoid system includes would be helpful. Your help is greatly appreciated, litgen is great!

**Comment by balos1** [comment]
The [SUNDIALS library now has Python bindings](https://sundials.readthedocs.io/en/latest/Python/index.html) built using nanobind and litgen. We use litgen to generate as much of the binding code as possible, and fill in the gaps with handwritten binding code. Overall, about 2/3rd of the binding code is generated with litgen!

**Reply by pthom** [reply]
@balos1 
Thanks for letting me know!I had a look at the nice doc and saw that SUNDIALS uses quite a lot of features from litgen. So, congrats!

**Reply by balos1** [reply]
Thanks for the useful tool @pthom !
